### ISSUE TITLE
Phase 1.3: Benchmark YOLO26 vs RT-DETRv2 latency and throughput

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#4247]
## Task

Measure inference latency and throughput for YOLO26 variants vs RT-DETRv2.

## Acceptance Criteria

- [ ] Measure single-image inference latency (ms) at 640x640 and 1920x1080
- [ ] Measure batch inference throughput (images/second) for batch sizes 1, 4, 8
- [ ] Measure GPU memory usage during inference
- [ ] Test on target GPU (RTX 4090 or similar)
- [ ] Include warmup iterations to exclude cold-start overhead

## Implementation Notes

```python
import time
import torch
from ultralytics import YOLO

model = YOLO('yolo26n.pt')
model.to('cuda:0')

# Warmup
for _ in range(10):
    model(dummy_image)

# Benchmark
torch.cuda.synchronize()
start = time.perf_counter()
for _ in range(100):
    model(image)
torch.cuda.synchronize()
latency_ms = (time.perf_counter() - start) / 100 * 1000
```

## Deliverables

* Latency comparison table
* VRAM usage comparison
* Throughput analysis at different batch sizes
